modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -39,6 +39,8 @@ def get_exception
         raise ::Common::Exceptions::ServiceError.new(
           detail: 'PtcpntIdA has open claims.'
         )
+      elsif record_not_found?
+        raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Record not found.')
```

Comment:
I think we are getting to the point where it makes sense to break up the error handling by service vs centralization.

[API-41059](https://jira.devops.va.gov/browse/API-41059)